documentation/asciidoc/topics/yaml/identities_batch.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: connect-secret
+type: Opaque
+stringData:
+  # Optional user for {brandname} metrics.
+  username: monitor
+  # Corresponding password for the "monitor" user.
+  password: password
+  # The key must be 'identities-batch'.
+  # The content is "user create" commands for the {brandname} CLI.
+  identities-batch: |-
+    user create user1 -p changeme -g admin
+    user create user2 -p changeme -g deployer
```

Comment:
@ryanemerson Does this look correct? I still get an exception in logs when I add this as a secret and configure the Helm chart to use it. I've double checked the secret has the "identities-batch" key.

Reply:
@oraNod What exception do you get?

Reply:
I know the issue now. If the user is creating the secret themselves then they must initialise the `metrics-users.properties` and `metrics-groups.properties` files in their batch OR remove the "metrics" securityRealm from `deploy.infinispan.server.security.securityRealms`.

I would just make the example 

```yaml
  identities-batch: |-
    user create user1 -p changeme -g admin
    user create user2 -p changeme -g deployer
    user create monitor -p password --users-file metrics-users.properties --groups-file metrics-groups.properties
```